src/components/Auth/Login/Login.react.js

```diff
@@ -112,26 +112,23 @@ class Login extends Component {
             });
             let accessToken = response.access_token;
             let uuid = response.uuid;
-            let state = this.state;
             let time = response.valid_seconds;
-
-            state.isFilled = true;
-            state.accessToken = accessToken;
-            state.success = true;
-            state.msg = response.message;
-            state.time = time;
-            this.setState(state);
-
             this.handleOnSubmit(email, accessToken, time, uuid);
             let msg = 'You are logged in';
-            state.msg = msg;
-            this.setState(state);
+            this.setState({
+              msg,
+              isFilled: true,
+              accessToken,
+              success: true,
+              msg: response.message,
+              time,
+            });
           } else {
-            let state = this.state;
-            state.msg = 'Login Failed. Try Again';
-            state.password = '';
-            state.showDialog = true;
-            this.setState(state);
+            this.setState({
+              msg: 'Login Failed. Try Again',
+              password: '',
+              showDialog: true,
+            });
           }
         }.bind(this),
```

Comment:
get rid of this **bind**

Reply:
Removed

Reply:
@AakashMallik fixed this  issue plsease merge the pr